pkg/operator/ceph/csi/util.go

```diff
@@ -104,15 +104,27 @@ func templateToDeployment(name, templatePath string, p templateParam) (*apps.Dep
 	return &ds, nil
 }
 
-func getToleration(provisioner bool) []corev1.Toleration {
+func getToleration(clientset kubernetes.Interface, namespace string, provisioner bool) []corev1.Toleration {
 	// Add toleration if any
 	tolerations := []corev1.Toleration{}
 	var err error
 	tolerationsRaw := ""
-	if provisioner {
-		tolerationsRaw = os.Getenv(provisionerTolerationsEnv)
-	} else {
-		tolerationsRaw = os.Getenv(pluginTolerationsEnv)
+	// TODO: export config map name
+	rookOperatorConfig, err := k8sutil.GetConfigMap(clientset, "rook-operator-config", namespace)
```

Comment:
how about giving an option to specify configmap name and namespace in operator.yaml?

Reply:
It will be a global config map in namespace that has rook operator.
we'll use this for other operator settings too.

Reply:
agreed, let's use a constant configmap name and require it to be in the same namespace as the operator